Dockerfile

```diff
@@ -11,6 +11,9 @@ WORKDIR $HOME/application
 # copy the entire application
 COPY . .
 
+# To handle 'not get uid/gid'
+RUN npm config set unsafe-perm true
+
```

Comment:
What was the issue here ? This doesn't seem right.

Reply:
It's removed after base image update